src/mailadm/bot.py

```diff
@@ -172,24 +123,100 @@ def is_admin_group_message(self, command: deltachat.Message):
         else:
             return False
 
-    def reply(self, text: str, reply_to: deltachat.Message, img_fn=None):
-        """The bot replies to command in the admin group
+    def handle_command(self, message: deltachat.Message):
+        """execute the command and reply to the admin."""
+        arguments = message.text.split(" ")
+        image_path = None
+
+        if arguments[0] == "/add-token":
```

Comment:
`len(arguments)` can't be 0, that would only be the case if `message.text` was empty. But this function is only called if `message.text.startswith("/")`, so that IndexError is already ruled out.